site/content/in-dev/unreleased/polaris-spark-client.md

```diff
@@ -0,0 +1,176 @@
+--conf spark.sql.catalog.<spark-catalog-name>.warehouse=<polaris-catalog-name> \
+--conf spark.sql.catalog.<spark-catalog-name>.header.X-Iceberg-Access-Delegation=vended-credentials \
+--conf spark.sql.catalog.<spark-catalog-name>=org.apache.polaris.spark.SparkCatalog \
+--conf spark.sql.catalog.<spark-catalog-name>.uri=<polaris-service-uri> \
+--conf spark.sql.catalog.<spark-catalog-name>.credential='<client-id>:<client-secret>' \
+--conf spark.sql.catalog.<spark-catalog-name>.scope='PRINCIPAL_ROLE:ALL' \
+--conf spark.sql.catalog.<spark-catalog-name>.token-refresh-enabled=true
+```
+Assume the released Polaris Spark client you want to use is `org.apache.polaris:polaris-iceberg-1.8.1-spark-runtime-3.5_2.12:1.0.0`,
+replace the `polaris-spark-client-package` field with the release.
+
+The `spark-catalog-name` is the catalog name you will use with Spark, and `polaris-catalog-name` is the catalog name used 
+by Polaris service, for simplicity, you can use the same name. 
+
+Replace the `polaris-service-uri`, `client-id` and `client-secret` accordingly, you can refer to
+[Using Polaris]({{% ref "getting-started/using-polaris" %}}) for more details about those fields.
+
+You can also start the connection by creating a Spark session, following is an example with PySpark:
+```python
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder
+  .config("spark.jars.packages", "<polaris-spark-client-package>,org.apache.hadoop:hadoop-aws:3.3.4,io.delta:delta-spark_2.12:3.3.1")
+  .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog")
+  .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,io.delta.sql.DeltaSparkSessionExtension")
+  .config("spark.sql.catalog.<spark-catalog-name>", "org.apache.polaris.spark.SparkCatalog")  
+  .config("spark.sql.catalog.<spark-catalog-name>.uri", <polaris-service-uri>)
+  .config("spark.sql.catalog.<spark-catalog-name>.token-refresh-enabled", "true")
+  .config("spark.sql.catalog.<spark-catalog-name>.credential", "<client-id>:<client_secret>")
+  .config("spark.sql.catalog.<spark-catalog-name>.warehouse", <polaris_catalog_name>)
+  .config("spark.sql.catalog.polaris.scope", 'PRINCIPAL_ROLE:ALL')
+  .config("spark.sql.catalog.polaris.header.X-Iceberg-Access-Delegation", 'vended-credentials')
+  .getOrCreate()
+```
+Similar as the CLI command, make sure the corresponding fields are replaced correctly.
+
+### Create tables with Spark
+After the Spark is started, you can use it to create and access Iceberg and Delta table like what you are doing before,
+for example:
+```python
+spark.sql("USE polaris")
+spark.sql("CREATE NAMESPACE IF NOT EXISTS DELTA_NS")
+spark.sql("CREATE NAMESPACE IF NOT EXISTS DELTA_NS.PUBLIC")
+spark.sql("USE NAMESPACE DELTA_NS.PUBLIC")
+spark.sql("""CREATE TABLE IF NOT EXISTS PEOPLE (
+    id int, name string)
+USING delta LOCATION 'file:///tmp/delta_tables/people';
```

Comment:
In general, we've switched to using "/var/tmp/" instead of "/tmp/" in Getting Started due to tmp dir's GC sometimes being quick to dump